checkstyle-tester/latest-projects-to-test-on.properties

```diff
@@ -0,0 +1,28 @@
+# List of GIT repositories to clone / pull for checking with Checkstyle
+# File format: REPO_NAME|[local|git|hg]|URL|[COMMIT_ID]|[EXCLUDE FOLDERS]
+# Please note that bash comments works in this file
+
+guava|git|https://github.com/google/guava|||
+
+spotbugs|git|https://github.com/spotbugs/spotbugs|||
```

Comment:
Let’s use `guava-latest` naming convention for all projects here to allow for easy copy paste and no conflicts on local.

Reply:
Done